### DISCUSSION TITLE
Chrome on iOS 16 not displaying AR button

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3979] [Q&A] [answered]
I have tested on iOS 16.1 and 16.1.1, and Chrome 107 and 108 and the AR button doesn't show. I have read some bug reports on this topic but I didn't find any fix or solution for this. Is there a solution/fix that I am missing?

**Comment by elalish** [comment]
Did you include the `ar` attribute? It would help if you post your code or a repro link.

**Reply by danilokrulj** [reply]
Tested it on this link: https://vizard.digital/promo/sofa-configurator/
You mean this attribute? 
![image_2022_11_30T14_51_01_923Z](https://user-images.githubusercontent.com/92090684/204828245-e50ce9db-8b6b-4c81-885a-85c497e648ee.png)

**Reply by danilokrulj** [reply]
A part of the code:
![image](https://user-images.githubusercontent.com/92090684/204832987-cdfa6b78-063d-41cc-9a61-fd08f0db47b0.png)

I also believe you can inspect the full code on the link where i tested.

**Comment by milesgreen** [comment, marked as answer]
iOS16 changed how AR feature detection worked in browsers other than Safari. As a result, you'll need to be using a newer version of ModelViewer (2.0.0 and up) to include this fix (https://github.com/google/model-viewer/pull/3784).

**Reply by danilokrulj** [reply]
So basically I should just up the module model-viewer@1.9.0 to @2.0?

**Reply by milesgreen** [reply]
Right. 2.0.0 or any of the 2.x.x releases since then. But make sure you're aware of the breaking changes in 2.0.0 and if they will impact your needs: https://github.com/google/model-viewer/releases/tag/v2.0.0

**Reply by danilokrulj** [reply]
Whoops, forgot to say thank you. Thank you! :)
